synapse/axon.py

```diff
@@ -1511,9 +1513,11 @@ async def jsonlines(self, sha256, errors='ignore'):
             try:
                 yield s_json.loads(line)
             except s_exc.BadJsonText as e:
-                logger.exception(f'Bad json line encountered for {sha256}')
-                raise s_exc.BadJsonText(mesg=f'Bad json line encountered while processing {sha256}, ({e})',
-                                        sha256=sha256) from None
+                # TODO: this feels like it should not be a log...
+                extra = self.getLogExtra(sha256=sha256, err=str(e))
+                logger.warning('Bad JSON line encountered.', extra=extra)
```

Comment:
```suggestion
                logger.warning('Bad JSON line encountered.', extra=extra, exc_info=e)
```
We should be able to pass in the exception as exc_info to get the proper traceback in the log but we'd still log this as a WARNING level exception. Then we can omit the `err=str(e)` above.

Reply:
Ah, cool! I didn't realize we could still pass exc_info. That also means we probably don't need the special / one-off `exc` key handling :+1: